### ISSUE TITLE
(invalid-read-syntax . "#") on running doom doctor

### ISSUE THREAD

**Issue by sasank89** [hlissner/doom-emacs#5092]
### 

- [X] I have searched [the FAQ](https://doomemacs.org/docs/faq.org) and [the documentation](https://doomemacs.org/docs/getting_started.org) for solutions.
- [X] I have searched [the issue tracker](https://github.com/hlissner/doom-emacs/issues) for similar issues (the closed issues too).
- [X] I have searched [the Discourse](https://discourse.doomemacs.org/docs) for any errors and solutions.
- [X] I can reproduce my issue on [the latest commit](https://github.com/hlissner/doom-emacs/commit) of Doom Emacs.
- [X] I have read [How to Debug Issues](https://discourse.doomemacs.org/how2debug) and found no solution.

### What did you expect to happen?

There were some random errors occasionally during the use of doom emacs. They dont really stop the functioning of it. I ran `doom doctor` to see if there was a code error. However, it returned an error as below. 

### What actually happened?

The error on running `doom doctor`

```
    x There was an unexpected error
      Message: Invalid read syntax
      Data: (invalid-read-syntax . "#")
      Backtrace:
        (read #<buffer  *load*-260766>)
        (eval-buffer #<buffer  *load*-260766> nil "/home/sasank/.doom.d/custom.el"
        (load-with-code-conversion "/home/sasank/.doom.d/custom.el" "/home/sasank/
        (load "~/.doom.d/custom.el" noerror t)
        (progn (load custom-file 'noerror (not doom-debug-mode)))
        (if custom-file (progn (load custom-file 'noerror (not doom-debug-mode))))
        (if no-config-p nil (maphash (doom-module-loader doom-module-config-file) 
        (progn (if doom-debug-p (progn (let ((inhibit-message (active-minibuffer-w
        (if init-p (progn (if doom-debug-p (progn (let ((inhibit-message (active-m
        (let* ((init-p (and t (condition-case e (let (file-name-handler-alist) (lo
```

### Steps to reproduce

1. Run `doom doctor` in terminal 

### System Information

https://pastebin.com/ZpBYASEn

**Comment by hlissner** [comment]
Please pastebin me your `~/.doom.d/custom.el`.

**Comment by sasank89** [comment]
[Pastebin Link](https://pastebin.com/wSR2bQCR)

Thank you so much for your quick reply.

**Comment by hlissner** [comment]
Delete your `custom.el` file. This was fixed in 5aea22ec3, but the fix is preventative. If your custom-file already saved `fixed-pitch` or `variable-pitch`, then you will see this error until you delete it.

As this was already fixed I will close this. If the issue resurfaces let me know.
